ui/admin/tests/acceptance/session-recordings/session-recording/channels-by-connection/channel-test.js

```diff
@@ -157,7 +170,7 @@ module(
         );
       await visit(urls.sessionRecording);
 
-      assert.dom('[data-test-manage-dropdown]').doesNotExist();
+      assert.dom(DROPDOWN_SELECTOR).doesNotExist();
```

Comment:
Non-blocking:
What was the reasoning for removing the `data-test-*` attribute and using HDS specific selectors?

Reply:
I removed it because it wasn't in the right place for the test I was writing but added those and removed the selectors